atomkraft/test/trace.py

```diff
@@ -55,4 +56,29 @@ def test_trace(trace: PathLike, reactor: PathLike, keypath: str):
             )
         )
     print(f"Executing {test_name} ...")
-    pytest.main(["-s", test_path])
+
+    report_dir = root / "reports" / test_name
+    report_dir.mkdir(parents=True)
+
+    logging_file = report_dir / "log.txt"
+
+    pytest_report_file = report_dir / "report.jsonl"
+
+    pytest_args = [
+        "--log-file-level=INFO",
+        f"--log-file={logging_file}",
+        f"--report-log={pytest_report_file}",
+    ]
+
+    if verbose:
+        pytest_args.append("-rP")
+        pytest_args.append("--log-cli-level=INFO")
+
+    pytest.main(pytest_args + [test_path])
+
+    default_nodes_dir = root / ".atomkraft" / "nodes"
+    report_nodes_dir = report_dir / "nodes"
+
+    shutil.copytree(default_nodes_dir, report_nodes_dir)
```

Comment:
Copy node data from `.atomkraft` directory to `report` directory.

Reply:
Probably put all this code in a separate function?